routes/api_v2.php

```diff
@@ -653,6 +656,15 @@ function () {
 //Route::put('file/{uuid}', [FilePropertiesController::class, 'update']);
 //Route::delete('file/{uuid}', [FilePropertiesController::class, 'destroy']);
 
+Route::prefix('audit-status')->group(function () {
+    Route::get('/', GetAuditStatusController::class);
+    Route::post('/', StoreAuditStatusController::class);
+});
+
+Route::prefix('site-polygon')->group(function () {
+    Route::get('/project/{uuid}', GetPolygonByProjectController::class);
```

Comment:
I'm confused why this endpoint is needed when we already have `/api/v2/projects/{uuid}/site-polygons` defined?